src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -630,11 +630,24 @@ MavlinkReceiver::handle_message_set_position_target_local_ned(mavlink_message_t
 	struct offboard_control_mode_s offboard_control_mode;
 	memset(&offboard_control_mode, 0, sizeof(offboard_control_mode));//XXX breaks compatibility with multiple setpoints
 
+	bool values_finite =
+		PX4_ISFINITE(set_position_target_local_ned.x) &&
+		PX4_ISFINITE(set_position_target_local_ned.y) &&
+		PX4_ISFINITE(set_position_target_local_ned.z) &&
+		PX4_ISFINITE(set_position_target_local_ned.vx) &&
+		PX4_ISFINITE(set_position_target_local_ned.vy) &&
+		PX4_ISFINITE(set_position_target_local_ned.vz) &&
+		PX4_ISFINITE(set_position_target_local_ned.afx) &&
+		PX4_ISFINITE(set_position_target_local_ned.afy) &&
+		PX4_ISFINITE(set_position_target_local_ned.afz) &&
+		PX4_ISFINITE(set_position_target_local_ned.yaw);
+
```

Comment:
I would argue that only the setpoints which are actually to be used (using the ignore mask) need to be finite.